src/components/CardsContainer/CardsContainer.component.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useState } from 'react';
+
+import { useFetchVideos } from '../../utils/hooks/useFetchVideos';
+import { Card } from '../Card/Card.component';
+import { VideoDetails } from '../VideoDetails/VideoDetails';
```

Comment:
By using the index file in each component to export the default component, you could change this like so:
```
import Card from '../Card';
import VideoDetails from '../VideoDetails';
```